include/erlcloud_ddb2.hrl

```diff
@@ -105,16 +109,12 @@
          replication_group :: undefined | [#ddb2_replica{}]
         }).
 
--record(ddb2_billing_mode_summary,
-       {billing_mode :: undefined | erlcloud_ddb2:billing_mode(),
-        last_update_to_pay_per_request_date_time :: undefined | date_time()}).
-
 -record(ddb2_provisioned_throughput_description,
         {last_decrease_date_time :: undefined | date_time(),
          last_increase_date_time :: undefined | date_time(),
          number_of_decreases_today :: undefined | integer(),
-         read_capacity_units :: undefined | pos_integer(),
-         write_capacity_units :: undefined | pos_integer()
+         read_capacity_units :: undefined | non_neg_integer(),
```

Comment:
I don't know if this is preferable to tweaking the test file. AWS console, for example, doesn't allow for IOPS of 0.

Reply:
Better to change tests if there is a case here—you really can't set this to 0.

Reply:
Fixed in b66b410.

Reply:
Are you sure? I still see it as `non_neg_integer()` in the diff here.

Reply:
I hadn't pushed it yet (I didn't know you'd be looking at this now :D). Sorry about that. It's pushed.